### ISSUE TITLE
Underscore used in "text/translateable" property

### ISSUE THREAD

**Issue by LukaszGrela** [adaptlearning/adapt_framework#1139]
In `course.json` [course.json#L89-L90](https://github.com/adaptlearning/adapt_framework/blob/master/src/course/en/course.json#L89-L90) the accessibility toggle btn labels are with underscore, but for consistency they should be without underscore.

**Comment by oliverfoster** [comment]
+1

**Comment by sarveshwar-gavhane** [comment]
PR : https://github.com/adaptlearning/adapt_framework/pull/1194
